src/Microsoft.AspNetCore.Blazor.Build/targets/All.props

```diff
@@ -3,5 +3,8 @@
 
   <PropertyGroup>
     <DefaultWebContentItemExcludes>$(DefaultWebContentItemExcludes);wwwroot\**</DefaultWebContentItemExcludes>
+
+    <!-- We can remove this after updating to newer Razor tooling, where it's enabled by default -->
+    <UseRazorBuildServer>true</UseRazorBuildServer>
```

Comment:
We set this to true in newer builds of the Razor Sdk. Might not need this to be explicitly set if you update the other